axolotl-web/src/pages/MessageList.vue

```diff
@@ -332,78 +330,56 @@ export default {
       }
     },
     scrollDown(){
-      if(document.getElementById("messageList-container")!=null)
-      document.getElementById("messageList-container").scrollTo(0,document.getElementById("messageList-container").scrollHeight);
+      var messageListContainer = document.getElementById("messageList-container");
+      if(messageListContainer != null) {
+        messageListContainer.scrollTo(0, messageListContainer.scrollHeight);
+      }
     },
     resetHeights(){
-      document.getElementById("messageInput").style.height="33px";
-    },
-    keyupHandler(e){
-      this.calcHeightsForInput(e);
+      document.getElementById("messageInput").style.height = "35px";
     },
     humanifyTimePeriod(time){
       if(time<60)
-      return time +" s";
+        return time +" s";
       else if(time<60*60)
-      return time/60+" m"
+        return time/60+" m"
       else if(time<60*60*24)
-      return time/60/60+" h"
+        return time/60/60+" h"
       else if(time<60*60*24)
-      return time/60/60/24+" d"
-
+        return time/60/60/24+" d"
       return time
     },
-    calcHeightsForInput(){
+    resizeMessageInput(){
       var el = document.getElementById("messageInput");
-      var c = document.getElementById("messageList-container");
-      if(window.innerHeight-c.clientHeight<200){
-        var scroll = el.scrollHeight;
-        if(scroll>150)scroll= 150;
-        if(Math.abs(this.lastCHeight-c.style.height)>10){
-          c.style.height = window.innerHeight-scroll-100+'px';
-          this.lastCHeight = c.style.height;
-        }
-        if(Math.abs(this.lastMHeight-el.style.height)>10){
-          el.style.height=el.scrollHeight+'px';
-          this.lastMHeight = c.style.height;
-        }
-      }
-      if(el.scrollHeight > el.clientHeight && el.scrollHeight<150){
-          el.style.height=el.scrollHeight+5+'px';
-          c.style.height = window.innerHeight-el.scrollHeight-100+'px';
-          if(document.body.scrollTop+550<document.body.scrollHeight)
-          window.scrollTo(0,document.body.scrollHeight);
-        }
-        // var scrollTop = window.pageYOffset || document.documentElement.scrollTop;
+      el.style.height = "5px";
```

Comment:
So, I don't get why, but [the internet](https://stackoverflow.com/questions/454202/creating-a-textarea-with-auto-resize) told me to first set an height, and then update it. They were using "auto" but that made the textarea two rows height when we only want one. A small px size works the same but made the transition ugly, so I had to drop them.